### DISCUSSION TITLE
Update without admin Rights?

### DISCUSSION THREAD

**Discussion by 007-fr** [Squirrel/Squirrel.Windows#1828] [Q&A] [answered]
Hello Squirrel community, 

I have an app that's installed in the normal location "C:\Program Files\App Name"
How can I push an update with Squirrel in this location without admin rights? The reason that I ask is because any time that I try and update my app with Squirrel, it prompts for admin credentials.  The goal that I am trying to get is to install the app on a machine and anyone who logs in can access the app. 

The way I have it now is that it works per user without prompting. 
![image](https://user-images.githubusercontent.com/66075770/187794126-ed08c439-ce5f-42e4-a623-f468767582f9.png)

**Comment by caesay** [comment, marked as answer]
It is not possible to have a Squirrel application installed in Program Files, or any other location that requires administrative privileges. If you would like to use Squirrel and you wish it to be available to all users, you will need to install the MSI deployment tool. [Read more about the machine-wide installer here](https://github.com/Squirrel/Squirrel.Windows/blob/develop/docs/using/machine-wide-installs.md).

**Reply by anaisbetts** [reply]
Refs #1608, #382
